tock/tock/templates/hours/reports_list.html

```diff
@@ -12,28 +12,33 @@ <h2>Regular Reports</h2>
 	<ul>
 		<li>Complete timecard data:
 			<a href="{% url 'reports:BulkTimecardList' %}">All</a>
-			<a href="{% url 'reports:BulkTimecardList' %}?after=2016-10-02">FY2017</a>
-            <a href="{% url 'reports:BulkTimecardList' %}?after=2017-10-01">FY2018</a>
+			{% for fy in fiscal_years %}
+				<a href="{% url 'reports:BulkTimecardList' %}?after={{ fy.start_date|date:'Y-m-d' }}&before={{ fy.end_date|date:'Y-m-d' }}">FY{{ fy.year }}</a>
```

Comment:
It's a minor thing, but inconsistency in our indentation hurts readability. Not asking for a change, just something to watch out for.